nav2_dwb_controller/dwb_core/include/dwb_core/dwb_local_planner.hpp

```diff
@@ -55,14 +57,17 @@ namespace dwb_core
  * @class DWBLocalPlanner
  * @brief Plugin-based flexible local_planner
  */
-class DWBLocalPlanner : public nav2_util::LifecycleHelperInterface
+class DWBLocalPlanner : public nav2_core::LocalPlanner
 {
 public:
   /**
    * @brief Constructor that brings up pluginlib loaders
    */
-  DWBLocalPlanner(
-    nav2_util::LifecycleNode::SharedPtr node, TFBufferPtr tf, CostmapROSPtr costmap_ros);
+  DWBLocalPlanner();
+
+  void initialize(
```

Comment:
you don't technically need both `virtual` and `override`, but its good to include `virtual` when you're overriding. I personally find that much clearer

Reply:
I think it helps to see that those functions with ```override``` are from the base class. Just a thought.